src/py21cmmc/core.py

```diff
@@ -273,11 +273,11 @@ def __init__(
             are required to specify the ionization field. Beyond this, the ionization field is specified directly from
             the perturbed density field.
         ctx_variables : list of str, optional
-            A list of strings, any number of the following: "brightness_temp", "init", "perturb", "xHI". These each
-            correspond to an OutputStruct which will be stored in the context on every iteration. Omitting as many as
+            A list of strings. The strings must correspond to any (pickleable) member of
+            :class:`py21cmfast.Coeval`. These will be stored in the context on every iteration. Omitting as many as
```

Comment:
Add the strings that users can choose from. Make life easy for us :)

Reply:
So the problem with adding the strings here is that the list might get out of date. What I might do is list a few here as typical suggestions, and refer the user to a function they can run to get the full current list of possibilities.